rdkit/Chem/ChemicalFeatures.py

```diff
@@ -8,8 +7,8 @@
 #  which is included in the file license.txt, found at the root
 #  of the RDKit source tree.
 #
-from rdkit.Chem.rdMolChemicalFeatures import *
 from rdkit.Chem.rdChemicalFeatures import *
```

Comment:
I believe that it's ok to do wildcard imports into a particular namespace.
The idea is that this file should be the only thing you need to import if you want to work with chemical features from inside of Python and that you shouldn't need to worry beyond that about namespaces.